code/Simulation/SimulationAgent.py

```diff
@@ -5,8 +5,9 @@
 from spade.behaviour import CyclicBehaviour
 from spade.message import Message
 
-from Simulation.Knowledge import Knowledge, KnowledgeInformation
+from Simulation import SimulationGraph
```

Comment:
Nie importujesz tu modułu zamiast klasy?

Reply:
Tak pycharm mi to sformatował i działało, sam nie wiem, zerknę w to

Reply:
to były i tak unused importy